tensorflow_serving/batching/batching_session.cc

```diff
@@ -79,6 +82,17 @@ TensorSignature TensorSignatureFromRunArgs(
   return signature;
 }
 
+std::vector<std::vector<std::pair<string, Tensor>>> GetTaskInputsVector(
```

Comment:
Please ensure that all functions & methods have a comment explaining (at least briefly) what they do.